### PR TITLE
Feature/88 ransack

### PR DESCRIPTION
**Pull request by jordanmaguire**
Use ransack instead of home-grown solution.

### CODE REVIEW

**Comment by jordanmaguire** [comment]
Fixes #87 and #88

File: app/views/admin/users/_user_results.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - # Only show the search options if there are users to search for
-- if params[:search].present? || @users.present?
-  = simple_form_for :search, method: :get do |s|
-    = s.input :search_term, label: "Email", placeholder: "Type email here", input_html: { value: (params[:search][:search_term] unless params[:search].blank?) }
+- if params[:q].present? || @users.present?
+  = search_form_for @ransack_query, method: :get, url: request.path do |s|
+    = s.input :email_cont, label: "Email", placeholder: "Type email here", as: :string, label_html: {class: "visually-hidden"}
```
Comment (pzi) [inline app/views/admin/users/_user_results.html.haml]: The label itself doesn't indicate that what is following is a search field. Maybe something like: "Search by Email"

Placeholder could be the example of <what goes in here>: "user@example.com".

When read out by a machine you'd have a clear label, an example of the expected input and the button that executes the search.

**Comment by Ruxton** [comment]
:+1:

File: app/views/admin/users/_user_results.html.haml (hunk 1 above)
Comment (jordanmaguire) [inline app/views/admin/users/_user_results.html.haml]: Can do

**Pull request merged**